pkg/templates/images/images.go

```diff
@@ -289,10 +289,10 @@ func optionalResources() map[Resource]map[string]string {
 		DigitalOceanCSISnapshotter:               {"*": "registry.k8s.io/sig-storage/csi-snapshotter:v6.2.1"},
 
 		// Hetzner CCM
-		HetznerCCM: {"*": "docker.io/hetznercloud/hcloud-cloud-controller-manager:v1.13.2"},
+		HetznerCCM: {"*": "docker.io/hetznercloud/hcloud-cloud-controller-manager:v1.15.0"},
 
 		// Hetzner CSI
-		HetznerCSI:                   {"*": "docker.io/hetznercloud/hcloud-csi-driver:2.2.0"},
+		HetznerCSI:                   {"*": "docker.io/hetznercloud/hcloud-csi-driver:2.3.2"},
```

Comment:
This should be prefixed with `v`, see https://hub.docker.com/layers/hetznercloud/hcloud-csi-driver/v2.3.2/images/sha256-0ed4f2f51459466f994c2ce6a566819e61c6dda26c1312d4e36172b74ea97d9d?context=explore

```suggestion
		HetznerCSI:                   {"*": "docker.io/hetznercloud/hcloud-csi-driver:v2.3.2"},
```

Reply:
Additional reference (thanks @xrstf): https://github.com/hetznercloud/csi-driver/pull/407/files